src/main/groovy/wooga/gradle/build/unity/UnityBuildPlugin.groovy

```diff
@@ -196,6 +214,21 @@ class UnityBuildPlugin implements Plugin<Project> {
                     t.secretsKey.set(secretsExtension.secretsKey)
                 }
 
+                project.tasks.register("newExport${baseName}", UnityBuildEngineTask) {
+                    UnityBuildEngineTask t ->
+                        t.group = "build unity"
+                        t.description = "exports gradle project for app config ${appConfigName}"
+                }
+
+                project.tasks.register("playerExport${baseName}", UnityBuildEnginePlayerTask) {
+                    UnityBuildEnginePlayerTask t ->
+                        t.group = "build unity"
+                        t.description = "exports player targeted gradle project for app config ${appConfigName}"
+                        t.appConfigFile.set(appConfig)
+                        t.secretsFile.set(fetchSecretsTask.flatMap({it.secretsFile}))
+                        t.secretsKey.set(secretsExtension.secretsKey)
+                }
```

Comment:
Ok what are these two tasks are for? One is the replacement for the current export tasks. What about the `newExport`?

Reply:
The newExport is the generic build, that isn't associated to the player build. I renamed it to "baseBuild${baseName}.